scripts/dts/extract_dts_includes.py

```diff
@@ -374,11 +368,7 @@ def extract_property(node_compat, yaml, node_address, prop, prop_val, names):
         def_label += '_' + label_override
 
     if prop == 'reg':
-        if 'partition@' in node_address:
-            # reg in partition is covered by flash handling
-            flash.extract(node_address, yaml, prop, def_label)
```

Comment:
Not fond of this part of the patch. Would prefer to keep all extract calls inside extract function.

Reply:
The problem is we call extract per property in most cases, and special case flash handling.  We really should be handling flash per node, not per property.

Reply:
Add comments in about doing per node handling vs per property.